Makefile

```diff
@@ -88,7 +88,7 @@ clean:  ## Clean all working/temporary files
 
 # DOCKER STUFF
 up: ## Start server using Docker
```

Comment:
For DEV, I like having Docker in the foreground. I'd prefer a different make command to run in the background.

Maybe we make an `up-d` command?